typify-impl/src/lib.rs

```diff
@@ -297,6 +306,12 @@ impl TypeSpaceSettings {
         self
     }
 
+    /// Replace a given schema with a named type. The given schema must precisely
+    /// match the schema in the type space, including fields like `description`.
+    /// The replacement named type is *not* included in the generated type space.
+    /// Typical usage is to map a schema definition to a builtin type or type
+    /// provided by a crate, such as `'rust_decimal::Decimal'`.
+    /// Additional conversions for the same schema are ignored.
```

Comment:
```suggestion
    /// Replace a given schema with a named type. The given schema must precisely
    /// match the schema from the input, including fields such as `description`.
    /// The replacement named type is *not* included in the generated type space.
    /// Typical usage is to map a schema definition to a builtin type or type
    /// provided by a crate, such as `'rust_decimal::Decimal'`.
    /// Additional conversions for the same schema are ignored.
```

```
    /// The replacement named type is *not* included in the generated type space.
...
    /// Additional conversions for the same schema are ignored.
```

Not sure what these mean.

Reply:
> The replacement named type is not included in the generated type space.

Let's delete this.

> Additional conversions for the same schema are ignored.

If the same schema is specified multiple times, only the first instance is honored.

Reply:
done